notebooks/advanced_RAG.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>The second point in the list seems unfinished. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Thank you! I completed it.</p>